sdks/python/apache_beam/transforms/trigger.py

```diff
@@ -337,12 +337,12 @@ def from_runner_api(proto, context):
             proto.after_processing_time
             .timestamp_transforms[0]
             .delay
-            .delay_millis))
+            .delay_millis) // 1000)
```

Comment:
We should change line 317 to
`        '', TimeDomain.REAL_TIME, context.get_current_time() + self.delay * 1000)
`

Reply:
I think the context clock timer unit is in seconds so I'm not sure we can use milliseconds conversion here.

Reply:
Sounds good. Then this would have been a bug when using milliseconds.